File: 30days/03/03.php
<?php

$handle = fopen ("php://stdin","r");
fscanf($handle,"%d",$N);

if ($N % 2 == 0) {
    if ($N >= 6 && $N <= 20) {
        echo "Weird";
    } else {
        echo "Not Weird";
    }
} else {
    echo "Weird";
}

?>

File: 30days/03/03.cpp
#include <iostream>

int main(){
    int N;
    std::cin >> N;
    if (N % 2 == 0) {
        if (N >= 6 && N <= 20) {
            std::cout << "Weird" << std::endl;
        } else {
            std::cout << "Not Weird" << std::endl;
        }
    } else {
        std::cout << "Weird" << std::endl;
    }
}
